google/cloud/storage/internal/curl_wrappers.cc

```diff
@@ -211,6 +211,15 @@ bool SslLockingCallbacksInstalled() {
 #endif  // GOOGLE_CLOUD_CPP_SSL_REQUIRES_LOCKS
 }
 
+CurlPtr MakeCurlPtr() {
+  auto handle = CurlPtr(curl_easy_init(), &curl_easy_cleanup);
+  // We get better performance using a slightly larger buffer (128KiB) than the
+  // default buffer size set by libcurl (16KiB).  We ignore errors because
+  // failing to set this parameter just affects performance by a small amount.
+  (void)curl_easy_setopt(handle.get(), CURLOPT_BUFFERSIZE, 128 * 1024L);
```

Comment:
quick question about `128 * 1024L`... do we want this to be `CURL_MAX_WRITE_SIZE` ?

Reply:
No, `CURLOPT_BUFFERSIZE` defaults to 
`CURL_MAX_WRITE_SIZE` and that is set to 16KiB (unless you override it at compile-time in libcurl).